tests/src/deprecated_field.rs

```diff
@@ -11,7 +11,12 @@ fn test_warns_when_using_fields_with_deprecated_field() {
     let message = deprecated_field::Test {
         not_outdated: ".ogg".to_string(),
         outdated: ".wav".to_string(),
+        test3: Some(deprecated_field::test::Test3::OneofOutdated(
+            ".mp3".to_string(),
+        )),
     };
+    #[allow(deprecated)]
+    let enum_ = deprecated_field::Test2::Outdated;
```

Comment:
I would like to see an additional test: generate a proto with only deprecated fields and write that to file. This has two purposes:
- It adds a example of all `deprecated` statements to the PR
- It actually tests that the `deprecated` flag is added to the fields.

The repo has examples for a test writing to file and asserting nothing changes: `test_generate_no_empty_outputs`

Reply:
Thanks for creating that test